src/cws_pattern.js

```diff
@@ -123,22 +133,41 @@ function get_crx_url(extensionID_or_url) {
     // $1 = m.0.r.p  // major.minor.revision.patch where minor is always 0 for some reason.
     // $2 = m
     // $3 =     r
-    var cr_version = /Chrome\/((\d+)\.0\.(\d+)\.\d+)/.exec(navigator.userAgent);
-    if (cr_version && +cr_version[2] >= 31 && +cr_version[3] >= 1609) {
+    var is_edge_browser = false;
+    // The user agent string for Edge browser contains 'Edg' followed by the browser version.
```

Comment:
While Edge is approximately a reskinned version of Chrome, the extension store is not. The Chrome Web Store is closed-source, and Microsoft created their own version from scratch.

The minimum to get a download to work in Microsoft Addons Store is the following:

`https://edge.microsoft.com/extensionwebstorebase/v1/crx?response=redirect&x=id%3Dcaoacbimdbbljakfhgikoodekdnlcgpk%26installsource%3Dondemand%26uc`

I don't see evidence of Microsoft serving platform-specific extension data, so this should be fine for now.

Since the backend of Google and Microsoft are independent, I suggest that you create a new function instead of modifying this function in several places.

Reply:
Added a single code block for the Microsoft Addons Store and reverted the multiple changes to this function.